odoo/orm/models.py

```diff
@@ -4354,14 +4381,14 @@ def _prepare_create_values(self, vals_list):
         :returns: new list of completed create values
         :rtype: dict
         """
-        bad_names = ['id', 'parent_path']
+        bad_names = {'id', 'parent_path'}
```

Comment:
Why ? It is just slower for no reason (and nondeterministic, in this case we don't care) . We only loop on it later and duplicate shouldn't exist here. Or I miss something ?